src/pages/Profile.tsx

```diff
@@ -132,154 +145,189 @@ const Profile = () => {
             },
           }
         );
-        const date = new Date(result.data.joinedDate);
-        const month = date.toLocaleString("default", { month: "long" });
-        const year = date.getFullYear();
-        const formattedDate = `${month} ${year}`;
         setProfileContents({
           ...result.data,
-          joinedDate: formattedDate,
         });
+        setLoading(false);
       } catch (error) {
-        console.error("Error fetching profile contents: ", error);
-      } finally {
+        console.log(error);
         setLoading(false);
       }
     };
     fetchProfileContents();
     window.scrollTo(0, 0);
   }, [username]);
 
+  // TODO: Make date utility functions or use a library for date formatting
+  const formatJoinedDate = (joinedDateString: string) => {
+    const joinedDateObj = new Date(joinedDateString);
+    const joinedMonth = joinedDateObj.toLocaleString("default", {
+      month: "long",
+    });
+    const joinedYear = joinedDateObj.getFullYear();
+    return `${joinedMonth} ${joinedYear}`;
+  };
+
+  const formatBirthDate = (birthDateString: string) => {
+    const birthDateObj = new Date(birthDateString);
+    return birthDateObj.toLocaleString("default", {
+      year: "numeric",
+      month: "long",
+      day: "numeric",
+    });
+  };
+
   return (
     <Layout
       middleContent={
-        <Box>
-          <Box style={styles.header}>
-            <IconButton onClick={() => navigate(-1)}>
-              <KeyboardBackspaceIcon color="secondary" />
-            </IconButton>
-            <Box>
-              <Typography sx={styles.displayName}>
-                {profileContents.displayName}
-              </Typography>
-              <Typography sx={styles.tweetCount}>
-                {profileContents.postCount} Tweets
-              </Typography>
-            </Box>
-          </Box>
+        <>
```

Comment:
Why don't we wrap the middleContent in a `Box`
```suggestion
        <Box>
            {...middle content...}
        </Box>
```

Reply:
I didn't wrap it in a `Box` because the `EditProfileModal` has to be there, and since modals don't really fit into the flow of the rest of the document I left it as a fragment.

With this being the case I ended up moving `EditProfileModal` outside of `Layout` entirely though because I think that it's not really part of the page. Let me know what you think about that